uavcan/node/IOStatistics.1.0.uavcan

```diff
@@ -0,0 +1,8 @@
+#
```

Comment:
This is not a high priority message to stabilize. Let's make it v0.1 or v0.0 to lower the work burden of initial stabilization?

Reply:
Agree. Demote to 0 for now.